docs/msource.md

```diff
@@ -0,0 +1,119 @@
+# MSource
+
+-----------
+
+## Point
+
+### Template structure
+
+### Description
+
+### Parameters
+
+### Dependencies
+
+### Products
+
+
+-----------
+
+## SeqConserved
+
+### Template structure
+
+One template argument `FImpl`, expected to be a fermion implementation.
+
+### Description
+
+Sequential source with an insertion of a conserved current $J_\mu$ using a `PropagatorField` $q(x)$ for $t_A\leq x_3 \leq t_B$:
```

Comment:
Could you please use S(x) instead of q(x) in the two first equations? We should avoid confusion between quark propagators and fields.